### PR TITLE
Fixed saving hash/bytes to a resource

### PR DESCRIPTION
**Pull request by roll**
- fix https://github.com/BCODMO/frictionless-usecases/issues/28

### CODE REVIEW

**Comment by roll** [comment]
@akariv 
Please take a look.

It seems to have fixed the `resource.hash/bytes` problem but I'm still not sure what's going on with `package.hash/bytes` (and what it is actually)

File: dataflows/processors/dumpers/file_dumper.py [hunk 1]
```diff
@@ -62,6 +62,7 @@ def handle_datapackage(self):
         temp_file_name = temp_file.name
         filesize = temp_file.tell()
         temp_file.close()
+        # TODO: it doesn't seem have any effect
```
Comment (akariv) [inline dataflows/processors/dumpers/file_dumper.py]: iirc it should add the size of the datapackage.json itself to the bytes count.

**Comment by akariv** [comment]
Bytes is for counting the number of bytes (approx.) the package weighs.
Hash is for saving the file in a hashed path (that is, re-saving the package in the same location btu with different data won't override the previous data)

**Comment by roll** [comment]
@akariv 
I didn't get that `inc` stands for `increment` in the first place.

I've removed the todos. It seems it works correctly.

**Comment by cschloer** [comment]
Just bumping this issue! I know its holiday season so no real rush but just want to keep this on our radar.

**Comment by roll** [comment]
I hope it's ready for a review

**Pull request merged**